docs/readmes/nms/federation.md

```diff
@@ -0,0 +1,55 @@
+---
+id: federation
+title: Federation
+hide_title: true
+---
+# Federated Networks
+
+Magma implements a low-friction, single point of integration between an MNO's mobile core and access gateways through federation networks. NMS enables management of the federated networks and their gateways.
+
+Magma has two important concepts on federation:
+
+### **Federation Network/** Federation Gateway (FGW)
+
+Federation network comprises of the federation gateways. The Federation Gateway integrates the Operators core network with using standard 3GPP interfaces to existing MNO components. It acts as a proxy between the Magma AGW and the operator’s network and facilitates core functions, such as authentication, data plans, policy enforcement, and charging.
+
+### **Federated LTE Network**
+
+Federated LTE network is the network containing AGWs which are managed through the federated networks. When configuring an integration with LTE nodes, it is necessary to link these two entities as described in the following sections.
+
+In the following sections, we will describe further on management of these federation network and federated LTE networks through NMS
+
+
+### Creating a Federation Network
+
+![adding networks](assets/nms/userguide/federation/adding_feg_network1.png)
+![adding networks](assets/nms/userguide/federation/adding_feg_network2.png)
+
+### Creating a Federated LTE Network
+![adding networks](assets/nms/userguide/federation/adding_feg_lte_network1.png)
+
+In the **Federated LTE** **Network**’s NMS page, the Federation config should contain the **Federation** **Network**’s network ID.
+![adding networks](assets/nms/userguide/federation/feg_association.png)
+
+### Configuring Federation Network
+
+Federation network can be managed like any other network in NMS. We can navigate to the newly created federation network by clicking on the appropriate network name in the network selector. Following image shows the top level view of a federation network. Here we display the currently configured federation gateways.
+
+![feg](assets/nms/userguide/federation/feg_overview1.png)
+Federation Gateway configuration page enables the operator to configure the server addresses for Gx, Gy, Swx, S6A and CSFB as shown below.
+![feg](assets/nms/userguide/federation/feg_configure1.png)
+
```

Comment:
what about adding a comment or a picture showing how Federation Network and Federated LTE Network relates?

Also on the menu below we have `feg_lte` and `feg`. Not sure what is the difference

![image](https://user-images.githubusercontent.com/16157139/116446990-5c818c80-a80c-11eb-8073-1ec21d041047.png)